apps/website/src/pages/_global.astro

```diff
@@ -74,6 +74,15 @@ const { content = {} } = Astro.props;
         font-size: var(--type-3);
       }
 
+      :where(h2:not(.astro-4EWMQMN4):not(left-sidebar)) {
+        border-top: 1px solid var(--color-line-2);
+        inline-size: 100%;
+        border-bottom: 1px solid rgb(245 245 245 / 0%);
+        border-left: 1px solid rgb(245 245 245 / 0%);
+        border-right: 1px solid rgb(245 245 245 / 0%);
```

Comment:
most of these declarations are unnecessary, you only need `border-block-start` and `padding-block-start`.

```suggestion
        border-block-start: 1px solid var(--color-line-2);
```

Reply:
fixed